modules/apps/data-engine/data-engine-test/src/testIntegration/java/com/liferay/data/engine/change/tracking/test/DEDataListViewTableReferenceDefinitionTest.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2023 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.data.engine.change.tracking.test;
+
+import com.liferay.arquillian.extension.junit.bridge.junit.Arquillian;
+import com.liferay.change.tracking.test.util.BaseTableReferenceDefinitionTestCase;
+import com.liferay.data.engine.service.DEDataListViewLocalService;
+import com.liferay.dynamic.data.mapping.model.DDMStructure;
+import com.liferay.dynamic.data.mapping.test.util.DDMStructureTestUtil;
+import com.liferay.journal.model.JournalArticle;
+import com.liferay.portal.kernel.model.change.tracking.CTModel;
+import com.liferay.portal.kernel.test.rule.AggregateTestRule;
+import com.liferay.portal.kernel.test.util.TestPropsValues;
+import com.liferay.portal.test.rule.Inject;
+import com.liferay.portal.test.rule.LiferayIntegrationTestRule;
+import com.liferay.portal.test.rule.PermissionCheckerMethodTestRule;
+
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+
+/**
+ * @author Gislayne Vitorino
+ */
+@RunWith(Arquillian.class)
+public class DEDataListViewTableReferenceDefinitionTest
+	extends BaseTableReferenceDefinitionTestCase {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerMethodTestRule.INSTANCE);
+
+	@Before
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+
+		_ddmStructure = DDMStructureTestUtil.addStructure(
+			group.getGroupId(), JournalArticle.class.getName());
+	}
+
+	@Override
+	protected CTModel<?> addCTModel() throws Exception {
+		return _deDataListViewLocalService.addDEDataListView(
+			group.getGroupId(), TestPropsValues.getCompanyId(),
+			TestPropsValues.getUserId(), "", _ddmStructure.getStructureId(),
+			"name", null, "");
```

Comment:
If you really need to pass in an empty string, use `StringPool.BLANK`. Otherwise, use `RandomTestUtil.randomString()` to get a random string. 

You can use `RandomTestUtil.randomLocaleStringMap()` to get a localized name map.